src/pconn.cc

```diff
@@ -348,33 +350,33 @@ PconnPool::key(const Comm::ConnectionPointer &destLink, const char *domain)
 }
 
 void
-PconnPool::dumpHist(StoreEntry * e) const
+PconnPool::dumpHist(std::ostream &yaml) const
 {
-    storeAppendPrintf(e,
-                      "%s persistent connection counts:\n"
-                      "\n"
-                      "\t Requests\t Connection Count\n"
-                      "\t --------\t ----------------\n",
-                      descr);
+    AtMostOnce explanation(
```

Comment:
Please update PR description. It does not match the current PR code. I recommend removing output sample from the PR description (and posting the updated sample as a PR comment).